resources/idlharness.js

```diff
@@ -1551,6 +1795,35 @@ IdlInterface.prototype.add_iterable_members = function(member)
            { name: "thisValue", idlType: {idlType: "any"}, optional: true}]}));
 };
 
+IdlInterface.prototype.test_to_json_operation = function(memberHolderObject, member) {
+    if (member.is_to_json_regular_operation()) {
```

Comment:
Yeah, I (vaguely) considered this at some point, but then dropped the ball. You're right. Will fix.